amr-wind/wind_energy/ABLBoundaryPlane.cpp

```diff
@@ -748,14 +748,12 @@ void ABLBoundaryPlane::read_header()
             m_in_data.define_plane(ori);
 
             const amrex::Box& minBox = m_mesh.boxArray(lev).minimalBox();
-            const auto& lo = minBox.loVect();
-            const auto& hi = minBox.hiVect();
 
-            amrex::IntVect plo(lo);
-            amrex::IntVect phi(hi);
+            amrex::IntVect plo(minBox.loVect());
+            amrex::IntVect phi(minBox.hiVect());
             const int normal = ori.coordDir();
-            plo[normal] = ori.isHigh() ? hi[normal] + 1 : -1;
```

Comment:
@michaeljbrazell I get the feeling this should be `plo[normal] = ori.isHigh() ? lo[normal] + 1 : -1;`. Is this a bug?